cypress/integration/user/generate.js

```diff
@@ -1,25 +1,13 @@
 describe('Generate users', () => {
   it('Should not have permissions when not logged in', () => {
     cy.visit('http://localhost:3000/user/generate')
-    cy.get('input[id="mail"]').click().type('test@wellcode.com')
-    cy.get('input[id="name"]').click().type('test name')
-    cy.get('input[id="password"]').click().type('test password')
```

Comment:
Let's use `.should('not.exist')` for some of the components instead of deleting them